custom-dashboards/monitoring.yml

```diff
@@ -0,0 +1,6 @@
+---
+- import_playbook: roles/openshift-ansible/playbooks/openshift-monitoring/config.yml
```

Comment:
Per the instructions above for `ansible-galaxy` and using `-p galaxy`, shouldn't this be `galaxy/openshift-ansible/...` ?